test/new-e2e/tests/sysprobe-functional/sysprobe_test.go

```diff
@@ -0,0 +1,128 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package sysprobefunctional
+
+import (
+	"flag"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	awshost "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/host"
+	"github.com/DataDog/datadog-agent/test/new-e2e/tests/windows"
+	windowsAgent "github.com/DataDog/datadog-agent/test/new-e2e/tests/windows/agent"
+	componentsos "github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
+	"github.com/stretchr/testify/require"
+)
+
+type vmSuite struct {
+	e2e.BaseSuite[environments.Host]
+}
+
+var (
+	kitchenDir string
+	testspath  string
+	reporoot   string
+	devMode    = flag.Bool("devmode", false, "run tests in dev mode")
+)
+
+func init() {
+	// Get the absolute path to the test assets directory
+	currDir, _ := os.Getwd()
+
+	reporoot, _ = filepath.Abs(filepath.Join(currDir, "..", "..", "..", ".."))
+	kitchenDir = filepath.Join(reporoot, "test", "kitchen", "site-cookbooks")
+	testspath = filepath.Join(kitchenDir, "dd-system-probe-check", "files", "default", "tests")
+}
+
+func TestVMSuite(t *testing.T) {
+	suiteParams := []e2e.SuiteOption{e2e.WithProvisioner(awshost.ProvisionerNoAgentNoFakeIntake(awshost.WithEC2InstanceOptions(ec2.WithOS(componentsos.WindowsDefault))))}
+	if *devMode {
+		suiteParams = append(suiteParams, e2e.WithDevMode())
+	}
+
+	e2e.Run(t, &vmSuite{}, suiteParams...)
+}
+
+func (v *vmSuite) TestTestSuite() {
+	v.T().Run("Works", v.testExample)
+}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why do you call the subtest from a `suite.T().Run()` ? Is it to eventually force the sequence of subtests ?